app/lib/searching.rb

```diff
@@ -24,7 +24,12 @@ class Searching
   ).freeze
 
   FIELDS = OTHER_FIELDS + ENHANCED_FIELDS + HEAD_FIELDS + SECONDARY_HEAD_FIELDS
-  OPTION_FIELDS = ENHANCED_FIELDS + HEAD_FIELDS + SECONDARY_HEAD_FIELDS + %w(content_id)
+  OPTION_FIELDS = ENHANCED_FIELDS + HEAD_FIELDS + SECONDARY_HEAD_FIELDS + %w(content_id).freeze
+
+  HOSTS = {
+    "production" => "http://rummager.dev.gov.uk",
```

Comment:
This is not production. Might be worth adding a dev option so that it can be used on local machine easily.